sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -2440,3 +2440,6 @@ VALUES
 (NULL, "SARS-COV-2 (COVID-19) vaccine, vector non-replicating, recombinant spike protein-ChAdOx1, preservative free, 0.5 mL ", "COVID-19 vaccine, vector-nr, rS-ChAdOx1, PF, 0.5 mL", 210, @codetypeid, '', 0, 0, '', '', '', 1);
 #EndIf
 
+#IfNotColumnType facility weno_id
+ALTER TABLE `facility` ADD `weno_id` VARCHAR(10) DEFAULT NULL;
+#EndIf
```

Comment:
This code block needs to be moved to the 6_0_0-to-6_1_0_upgrade.sql script (i will migrate it to the patch.sql script in future when add this code to next 6.0.0 patch)